sdk/core/core/inc/az_config.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef _az_CONFIG_H
+#define _az_CONFIG_H
+
+#include <az_span.h>
+
+#include <_az_cfg_prefix.h>
+
+/*
+ *
+ */
+enum {
+  AZ_HTTP_REQUEST_URL_BUF_SIZE = 2* 1024,
+  AZ_HTTP_REQUEST_BODY_BUF_SIZE = 1024,
+  AZ_HTTP_REQUEST_HEADER_COUNT = 10,
+  AZ_HTTP_REQUEST_HEADER_BUF_SIZE = AZ_HTTP_REQUEST_HEADER_COUNT * sizeof(az_pair),
+  AZ_HTTP_RESPONSE_BUF_SIZE = 3 * 1024
```

Comment:
Who uses this? I think it would only ever be used by customer code (and maybe client secret credential now after today;s discussion), right? If it's only used by customer code, then we should not have this enum - it is the customer's issue. If we only need this for AAD, then maybe we should just make this internal for that specific scenario and not have a public enum.